### PR TITLE
Update chappie2.0 with latest changes from develop

### PR DESCRIPTION
**Pull request by afjlambert**

### CODE REVIEW

**Comment by afjlambert** [comment]
Chappie2.0 should be kept up to date with changes from develop. @robinpiets can you look at the merge conflict?

File: apps/client/sttr_build/config.js [hunk 1]
```diff
@@ -50,6 +50,16 @@ module.exports = {
...
+    zonwering: [
+      {
+        id: "TuP6ido9xpJSSxjLi",
+        name: "Zonwering, rolhek, luik of rolluik bouwen",
+      },
+      {
+        id: "AkxinYRNNo679qi6T",
+        name: "Zonwering, rolhek, luik of rolluik monument",
+      },
+    ],
```
Comment (robinpiets) [inline apps/client/sttr_build/config.js]: Should `monument` not be the first object in the array or does it not matter?

**Review by robinpiets — APPROVED** [review]
🔥

**Review by robinpiets — CHANGES_REQUESTED** [review]
Bump helmet from 3.23.3 to 4.0.0 (#340) creates Netlify errors....

image

Edit: `options.permittedPolicies` is a string that must be "none", "master-only", "by-content-type", or "all"

**Review by robinpiets — APPROVED** [review]

**Comment by robinpiets** [comment]
Do we want to wait for #365 #367 ? @afjlambert

**Pull request merged**